pages/cart-management/cart-management.html

```diff
@@ -67,20 +66,24 @@ <h2 class="text-center">My Cart(4)</h2>
             <div class="card-image-container">
               <img class="responsive-img rounded-top-corner-img" src="/assets/curiosity-mug-card-img.webp" alt="card-img" />
             </div>
-            <div class="card-info-container">
-              <div class="card__titles">
-                <h3 class="card-heading">Curiosity Rover 11 oz Ceramic Mug</h3>
-                <h2 class="card-sub-heading">₹ 1317 <span class="text-line-through text-gray text-md">₹ 2,317</span></h2>
+
+            <div class="horizontal-card-content">
+              <div class="card-info-container text-left">
+                <span class="text-bold card-heading">CognitiveSurplus</span>
+                <span class="card-sub-heading text-gray">Curiosity Rover 11 oz Ceramic Mug Lorem ipsum dolor sit amet.</span>
+                <div class="price">
+                  <span class="card-sub-heading text-md">₹ 1,317 </span><span class="text-line-through text-xs text-gray">₹ 2,317</span>
+                </div>
               </div>
               <div class="order-counter">
                 <p>Quantity:</p>
```

Comment:
Can we use span instead of p tag for quantity ? as that seems more accessibility friendly

Reply:
You are right it should be `<span>`. Will change it now. Thanks for the review.

Reply:
> Can we use span instead of p tag for quantity ? as that seems more accessibility friendly

You are right it should be <span>. Will change it now. Thanks for the review.